test/runtests.jl

```diff
@@ -25,8 +25,8 @@ for test in [
         failed_tests += 1
         println(output)
         println(outbuf)
-        println(UTF8String(output))
-        println(UTF8String(outbuf))
+        println(String(output))
+        println(String(outbuf))
```

Comment:
I don't think this is supported. (This is in the error path so it may not show up in the CI)